test/test_db/test_db.py

```diff
@@ -36,18 +33,18 @@ def test_trigger_update_updated_date(self):
 
         # Insert test record
         test_case_name = TestDb.get_test_case_name()
-        insert_query = "INSERT INTO base.data_source_type (name) VALUES ('{}');".format(test_case_name)
+        insert_query = f"INSERT INTO base.data_source_type (name) VALUES ('{test_case_name}');"
         self.connection.execute(insert_query)
         self.connection.commit()
 
         # Update test record
         test_case_name_updated = TestDb.get_test_case_name()
-        update_query = "UPDATE base.data_source_type SET name = '{}' WHERE name = '{}';".format(test_case_name_updated, test_case_name)
+        update_query = f"UPDATE base.data_source_type SET name = '{test_case_name_updated}' WHERE name = '{test_case_name}';"
         self.connection.execute(update_query)
         self.connection.commit()
 
         # Get updated record
-        select_query = "SELECT created_date, updated_date FROM base.data_source_type WHERE name = '{}';".format(test_case_name_updated)
+        select_query = f"SELECT created_date, updated_date FROM base.data_source_type WHERE name = '{test_case_name_updated}';"
```

Comment:
Use single quotes and escape everywhere in this file.

Reply:
Done